ngshare/version.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.0.1'
```

Comment:
Can we bump this to like 0.2.0 or something? I used up a bunch of lower version numbers during testing. Anything before 1.0.0 is probably fine

Reply:
Maybe you can go ahead and change it. I do not care that much about version number.